R/get_meta_url.R

```diff
@@ -22,35 +20,35 @@ get_meta_fld <- function(slug) {
 
   # TODO: consider storing this as a table. Slugs in one column, URLs in the other
```

Comment:
It would be my suggestion to put this as a datafile in `inst/extdata`; hardcoded URLs like this are easy to overlook when updates are made.

Reply:
Good idea, I may do this now. what does `inst` mean here?

Reply:
`/inst/extdata` is the folder that is conventionally used for data files that are used inside the package, but are not `.Rda` or `.Rdata` files.

Reply:
thanks for the explanation!